articles/openstack/ostack-how-basic-heat-template.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How to create and deploy a basic heat template  | UKCloud Ltd
+description: Instructions on how to deploy a basic heat template
+services: openstack
+author: Bryce Nicholls
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create and deploy a basic heat template 
+toc_fullpath: How To/ostack-how-static-ip.md
+toc_mdlink: ostack-how-basic-heat-template.md
+---
+
+# How to create and deploy a basic heat template 
+
+## Overview
+
+This is a basic heat orchestration template create a heat stack.
+
+This template will create the following and connect them up.
+- A network
+- A static IP
+- A router
+- An instance
+- A floating IP
+- A volume
+
+## The heat template
+
+> [!NOTE]
+> Make sure you check the indentation and file extension. It needs to be in yaml format.
```

Comment:
Suggest:
The template must be in yaml format and be saved with the .yaml file extension. You should also make sure that you use the correct indentation.